text/0000-idempotent-install.md

```diff
@@ -0,0 +1,68 @@
+- Start Date: 2016-12-13
+- RFC PR: (leave this empty)
+- Yarn Issue: (leave this empty)
+
+# Summary
+
+`yarn install` should be idempotent.
+
+Ideally, the result of `yarn install` would not be statefully dependent on the contents of an existing `node_modules`, and would ensure the resulting `node_modules` is identical regardless of whether there is an existing `node_modules` or not. This seems very inline with the spirit of yarn's goal of deterministic builds (same `node_modules` independent of whether `node_modules` exists or the version of node that generated it).

```

Comment:
Based on your comment & several 👍, I'll assume this rfcs is green-lighted. Is it safe to assume a proper PR / implementation of this rfcs will be accepted?